modules/i3s/src/lib/parsers/parse-i3s.js

```diff
@@ -31,9 +31,20 @@ export function normalizeTileData(tile, options, context) {
 
 export function normalizeTileNonUrlData(tile) {
   scratchCenter.copy(tile.mbs);
-  const centerCartesian = Ellipsoid.WGS84.cartographicToCartesian(tile.mbs.slice(0, 3));
+  const sphere = [
+    ...Ellipsoid.WGS84.cartographicToCartesian(tile.mbs.slice(0, 3)), // cartesian center of sphere
```

Comment:
Nit: I prefer calculating this value on a separate line. The intermediary variable name makes the code more self documenting.